pkg/ocm/core.go

```diff
@@ -112,7 +112,8 @@ func (o *OCMInstance) ListManagedClusters(clusters []*v1.Cluster) []*v1.Cluster
 	var clusterList []*v1.Cluster
 
 	for _, cluster := range clusters {
-		if cluster.Product().ID() != "ocp" {
+	    cluster_id := cluster.Product().ID()
+		if cluster_id != "ocp" && cluster_id != "aro" {
```

Comment:
```suggestion
	    cluster_product_id := cluster.Product().ID()
		if cluster_id != "ocp" && cluster_id != "aro" {
```

iiu go correctly you can do something like this pseudo code:
```
umanamged_products := []str{"ocp", "aro"}
 cluster_product_id := cluster.Product().ID()
		if ! slices.Contains(umanamged_products, cluster_product_id) {
```

Reply:
done